app/controllers/project_groups_controller.rb

```diff
@@ -48,7 +48,7 @@ def update
       @project_group.options = ::Filters::HudFilterBase.new(user_id: current_user.id, project_type_numbers: []).update(filter_params).to_h
       @project_group.save
       users = user_params[:users]&.reject(&:empty?)
-      @project_group.update_access(users.map(&:to_i)) if users.present?
+      @project_group.update_access(users.map(&:to_i))
```

Comment:
This fixes an unrelated bug. This check made it so you could never delete the last access group, because `users` would be empty. As a result, the "Delete Group" button never appeared, so project groups effectively couldn't be deleted at all. Let me know if I'm missing something here @eanders

Reply:
Oh good point, I'll check that. We should only update access if the `users` param is present (even if it is empty).